tests/unit/utils/test_misc.py

```diff
@@ -1,56 +1,37 @@
+import pytest
```

Comment:
It would be great if the unittests were not dependent to pytest, and in particular, to the pytest fixtures we declare. We want the unittest to run with the less dependencies / arguments possible. `root_pytest_dir` is a very simple fixture, but it is a first step of tying together two worlds we do not want to mix.

Reply:
Reverted the fixture usage